safe-defaults/tmpfiles-enable.conf.in

```diff
@@ -13,6 +13,8 @@ L @SYSCONFDIR@/opt/chrome/policies/managed/safe-search.json - - - - @DATAROOTDIR
 D /run/flatpak/extension/org.chromium.Chromium.Policy.safe-search 0755 - - -
 D /run/flatpak/extension/org.chromium.Chromium.Policy.safe-search/@FLATPAK_ARCH@ 0755 - - -
 L /run/flatpak/extension/org.chromium.Chromium.Policy.safe-search/@FLATPAK_ARCH@/1 - - - - @DATAROOTDIR@/eos-safe-defaults/chrome
+D /var/local/share/ca-certificates 0755 - - -
+L /var/local/share/ca-certificates/Cisco_Umbrella_Root_CA.crt - - - - @DATAROOTDIR@/eos-safe-defaults/Cisco_Umbrella_Root_CA.cer
```

Comment:
Although `/usr/local` resolves to `/var/local` on Endless, I think it would be better to use `/usr/local` unless that causes problems with systemd.

Reply:
sure